ts/components/bottomSheet/BottomSheetHeader.tsx

```diff
@@ -50,7 +39,9 @@ export const BottomSheetHeader: React.FunctionComponent<Props> = ({
   onClose
 }: Props) => (
   <View style={styles.row}>
-    <H3>{title}</H3>
+    <View style={{ flex: 1 }}>
```

Comment:
Maybe is better:
`<View style={IOStyles.flex}>`

Reply:
Good catch! I just lost it during refactoring 😄